ec2/al2-mutable-private.yaml

```diff
@@ -216,6 +218,47 @@ Mappings:
       Map: RegionMap202002071
     '20191217.0':
       Map: RegionMap201912170
+  RegionMap20200917:
+    'af-south-1':
+      AMI: 'ami-0cec12e29ebe3f0d5'
```

Comment:
Why don't you replace `AMI` with the version? A CloudFormation map is already a map with keys on two levels.

Reply:
It's just easier to maintain. Copy the whole block instead of inserting a new value for every region key into an existing map. But feel free to change in a new PR. This is just about updating AL2 :)